src/Map/map.web.js

```diff
@@ -52,8 +52,8 @@ export const getMap = ({
       }}
     >
       {filteredMarkers &&
-        filteredMarkers.map((marker) => (
-          <View lat={marker.lat} lng={marker.lng} onClick={marker.onPress}>
+        filteredMarkers.map((marker, index) => (
+          <View lat={marker.lat} lng={marker.lng} onClick={marker.onPress} key={`marker ${index}`}>
```

Comment:
Using a list index as the `key` is [generally discouraged](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318) when there's an alternative available. Would it make sense to use something like the lat/long values here instead?

Reply:
I can make this change. I was under the impression that using the index on it's own is discouraged.

Reply:
That's usually the case that's discussed but my understanding is it's technically any key that is strictly a derivative of the index as all the same performance issues and potential bugs apply.